src/hooks/api-queries.ts

```diff
@@ -321,6 +321,29 @@ export const useListTasksQuery = ({ projectId, milestoneId, first, max }: ListTa
   });
 };
 
+/**
+ * Lists tasks affected by a change proposal
+ *
+ * @param changeProposalId change proposal id
+ */
+export const useListChangeProposalAffectedTasksQuery = (changeProposalId?: string) => {
+  const { tasksApi } = useApi();
+  const { t } = useTranslation();
+
+  return useQuery({
+    queryKey: ["changeProposalTasks", changeProposalId],
```

Comment:
It would be better to directly use useListTasks, as it would then share the correct query key and parameters.